src/components/KEmptyState/KEmptyState.vue

```diff
@@ -1,143 +1,167 @@
 <template>
-  <section
-    class="empty-state-wrapper"
-    :class="{ 'is-error': isError }"
+  <div
+    class="k-empty-state"
+    :class="[iconVariant]"
   >
-    <div class="empty-state-title">
-      <div
-        v-if="isError || icon"
-        class="k-empty-state-icon card-icon"
-        :class="{ 'warning-icon': isError }"
-      >
-        <KIcon
-          :color="isError ? iconColor || `var(--kui-color-text-neutral-strongest, ${KUI_COLOR_TEXT_NEUTRAL_STRONGEST})` : iconColor"
-          :icon="icon ? icon : 'warning'"
-          :secondary-color="isError ? iconSecondaryColor || 'currentColor' : iconSecondaryColor"
-          :size="iconSize"
-        />
-      </div>
-      <div
-        v-if="$slots.title"
-        class="k-empty-state-title-header"
-      >
-        <slot name="title" />
-      </div>
-    </div>
     <div class="empty-state-content">
-      <div
-        v-if="$slots.message"
-        class="k-empty-state-message"
-      >
-        <slot name="message" />
+      <div class="empty-state-icon">
+        <slot name="icon">
+          <component
+            :is="getEmptyStateIcon"
+            :color="getIconColor"
+            :size="KUI_ICON_SIZE_60"
+          />
+        </slot>
       </div>
+      <span
+        v-if="title"
+        class="empty-state-title"
+      >
+        {{ title }}
+      </span>
       <div
-        v-if="$slots.cta || (!ctaIsHidden && ctaText)"
-        class="k-empty-state-cta"
+        v-if="message || $slots.default"
+        class="empty-state-message"
       >
-        <slot name="cta">
-          <KButton
-            appearance="primary"
-            size="small"
-            @click.prevent="() => handleClick && handleClick()"
-          >
-            {{ ctaText }}
-          </KButton>
+        <slot name="default">
+          <p>
+            {{ message }}
+          </p>
```

Comment:
Kinda similar to the above:

Q: Is anyone ever likely to have a 2 paragraph empty message? Or some other semantic detail that wrapping it with a `p` could be problematic?

edit: I think this might be problematic in the case of KAlert (which also enforces a single `p` here). Specifically our KAlert message in places is a `ul` which is not [Phrasing content](https://developer.mozilla.org/en-US/docs/Web/HTML/Content_categories#phrasing_content) and is therefore not allowed within a p tag:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p#technical_summary

I think, as a very general point, these types of semantic elements like `p` and `h*`s shouldn't be used inside the majority of reusable components as it makes them less reusable. Just to stress this is a very general point, i.e. we should think twice when adding them. and I guess there will always be exceptions.

Reply:
Oh whoops! Ignore me 😅 I think I read this wrong, we can overwrite the p with the slot.